### ISSUE TITLE
Entering password locally to unlock screen makes the password appear on remote computer

### ISSUE THREAD

**Issue by andy5995** [rustdesk/rustdesk#1200]
**Describe the bug you encountered:**
I was working with someone, connected remotely with rustdesk. I locked the screen with ctrl-alt-del. When I came back a few minutes later, I entered my password to unlock (xscreensaver) my screen. After unlocking, I could see my password appeared on the remote computer about the same time my friend was entering text into an IRC channel, which resulted in my local password being sent to an IRC channel.

---

**How did you install `RustDesk`?**

AUR

---

**RustDesk version and environment**

1.1.9-2 on XFCE4

Manjaro 21.3.6

**Comment by rustdesk** [comment]
@asur4s

**Comment by andy5995** [comment]
I experienced a (probably) related problem last week. Using the docker rustdesk server which I pulled about a month ago.

Two computers connecting to rustdesk server on my LAN.

I was working primarily on my laptop, from where the rustdesk client was connected to my desktop, both on my LAN (the server running on my desktop).

I had my IDE, Geany, open on my desktop. On my laptop, I was working in an XFCE terminal, issuing many commands over a period of about 10 minutes. When I clicked back over to view my desktop screen (from within my rustdesk client on my laptop), I saw that all of the commands I entered into the terminal had been inserted into the open document in Geany. A different line for each each command, just as I had entered into the terminal.

**Comment by rustdesk** [comment]
I believe https://github.com/rustdesk/rustdesk/releases/tag/nightly solves this problem.

**Comment by andy5995** [comment]
I experienced this problem again. Me and a friend are both on Manjaro using rustdesk-bin 1.20 from the AUR. The rustdesk ID server we're connected to is running on my local network, last pulled 4 months ago.

1. I was connected to my friend over the Internet.
2. From the same client, I also connected to my desktop over my LAN (laptop to desktop using a crossover cable)
3. I opened up a terminal on my remote desktop
4. I typed into the terminal and my cursor didn't move at all, but the characters appeared in my friend's terminal instead.
